action.yml

```diff
@@ -47,7 +47,15 @@ runs:
           if [[ ${{ inputs.cosign-release }} == 'v0.6.0' ]]; then
             # v0.6.0's linux release has a dependency on `libpcsclite1`
             sudo apt-get update -q
```

Comment:
Question: do we need to run the update even if the dependency exist ? That is my main concern

Reply:
if the dependency exist i don't think we need to update

Reply:
yes you are right, I fixed it.

Reply:
thank you so much, folks @cpanato @hectorj2f